libevmasm/Ethdebug.cpp

```diff
@@ -36,53 +36,56 @@ Json programInstructions(Assembly const* _assembly, LinkerObject const& _linkerO
 			solUnimplementedAssert(instruction.type() != VerbatimBytecode, "Verbatim bytecode is currently not supported by ethdebug.");
 	}
 
-	solAssert(_linkerObject.codeSectionLocations.size() == 1);
-	solAssert(_linkerObject.codeSectionLocations[0].end <= _linkerObject.bytecode.size());
-	Json instructions = Json::array();
-	for (size_t i = 0; i < _linkerObject.codeSectionLocations[0].instructionLocations.size(); ++i)
+	if (_linkerObject)
 	{
```

Comment:
You should do this to get saner diff :)
```suggestion
	if (!_linkerObject)
		Json::array();
```

Reply:
But also why is it even called when the object is null? Wouldn't make more sense to just skip it?